### PR TITLE
Add documentation for the ColumnsEditor UI component

### PR DESCRIPTION
**Pull request by serhiyzhovnir**
## Purpose of this pull request

This pull request (PR) adds the documentation for the ColumnsEditor UI component.

## Affected DevDocs pages

The new page will be created with the URL: https://devdocs.magento.com/guides/v2.3/ui_comp_guide/components/ui-columns-editor.html

## Links to Magento source code

-  https://github.com/magento/magento2/blob/2.3/app/code/Magento/Cms/view/adminhtml/ui_component/cms_page_listing.xml#L118-L126
-  https://github.com/magento/magento2/blob/2.3/app/code/Magento/Ui/view/base/web/js/grid/listing.js#L45-L51
-  https://github.com/magento/magento2/blob/2.3/app/code/Magento/Ui/view/base/web/js/grid/editing/editor.js

whatsnew
Added a topic for the [ColumnsEditor UI component](https://devdocs.magento.com/guides/v2.3/ui_comp_guide/components/ui-columns-editor.html).

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.3/ui_comp_guide/components/ui-columns-editor.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+---
+group: ui-components-guide
+title: ColumnsEditor component
+contributor_name: Atwix
+contributor_link: https://www.atwix.com/
+---
+
+The ColumnsEditor [UI component](https://glossary.magento.com/ui-component) is an [extension](https://glossary.magento.com/extension) for [Columns]({{ page.baseurl }}/ui_comp_guide/components/ui-columns.html) component, allowing users to select and edit grid records data.
```
Comment (dobooth) [inline src/guides/v2.3/ui_comp_guide/components/ui-columns-editor.md]: ... for the Columns component, allowing...

**Comment by serhiyzhovnir** [comment]
Hi @dobooth
This PR was improved, please check it again.
Thank you!

**Comment by serhiyzhovnir** [comment]
Hi @dobooth
I've added requested changes.
Thanks.

**Comment by rogyar** [comment]
Hi @serhiyzhovnir. Could you check the conflicting files, please? 

Thank you!

**Comment by serhiyzhovnir** [comment]
Hi @rogyar @dobooth 
The conflict is fixed.
Thank you!

**Comment by dobooth** [comment]
running tests

**Pull request merged**